tests/ui/test_non_admin_user.py

```diff
@@ -43,6 +44,7 @@ def finalizer():
 
     @ui
     @tier2
+    @skipif_ibm_cloud
```

Comment:
We will need to have more specific marker for ROKS (managed deployment type)

  config.ENV_DATA["deployment_type"].lower() != "managed" and you need to add the ibm platform as well

Reply:
added new marker, thanks